css/index.css

```diff
@@ -1 +1,315 @@
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+    align-items: center;
+    /*TRBL*/
+    margin: 0 /*82px*/9.31% 0 0;
+}
+
+.cta .cta-text button {
+    border: 1px double black;
+    font-size: 1.6rem;
+    background: white;
+    width: 170px;
+    height: 32px;
+    margin-top:/* 10px*/1.13% ;
+    cursor: pointer;
+}
+/* if I don't have this .cta img block here, get started button gets messy*/
+.cta img {
+    max-width: 100%;
+    margin-right: 1%;
+}
+.cta .cta-text button:hover {
+    background: black;
+    color: white;
+}
+
+/* Main Content Styles */
+
+.main-content {
+   border-top: 2px solid black; 
+   border-bottom: 2px solid black;
+   margin: 75px 0 35px 0;
+   padding: 35px 0;
+   font-size: 1.6rem;
+   line-height: 1.3rem;
+}
+
+.main-content .top-content {
+    display: flex;
+  justify-content: space-around;  
+}
+
+.main-content .middle-img {
+    max-width:100%;
+    display: block;
+    margin:/* 30px*/3.4%  auto;
+}
+
+.main-content .bottom-content {
+    display: flex;
+    justify-content: space-around;
+}
+
+.main-content .bottom-content p {
+    padding-right:/* 20px*/2.27% ;
+}
+
+/* Contact Styles */
+
+.contact p {
+    margin: 15px 0;
+}
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+
+/* tablet*/
+@media (max-width:800px){
+    .container {
+    width: 100%;
+    background: orange;
+    
+  }
+  header {
+      flex-wrap: wrap;
+  }
+    header nav {
+        width: 100%;
+        padding: 38px 3%;
+           }
+   
+   header .logo {
+  display: block;
+  order: -1;
+  margin: 30px auto 0;
+}
+        .cta img{
+    display: none;
+       }
+
+       .cta {
+           justify-content: center;
+           margin-top: 35px;
+               }
+               .cta .cta-text h1 {
+                   letter-spacing: 0;
+                   margin-bottom: 15px;
+                                  }
+  .main-content .middle-img {
+       width: 100%;
+       margin: 10px auto 25px;
+    }
+
+    .main-content {
+       margin: 75px 2% 35px;
+    }
+
+    .text-content {
+        margin: 0 10px;
+    }
+
+    .contact {
+        margin: 20px 2%;
+    }
+                                
+}
+/* mobile*/
+@media (max-width:500px){
+    .container {
+    background:red;
+    
+    }
+  header{
+      justify-content: center;
+  }
+    header nav {
+        flex-direction: column;
+        margin: 0;
+              
+        }
+
+        header nav a {
+            width: 100%;
+            display: block;
+            text-align: center;
+            padding: 20px 0;
+            font-size: 2.0rem;
+            border-bottom: 1px solid gray;
+        }
+
+        header nav a:first-child {
+        border-top: 1px solid gray;
+    }
+
+    header nav a:hover {
+        background: gray;
+    }
+    
+    header .logo {
+    margin: 20px 0;
+}
+.cta{
+    margin-top: 50px;
+        flex-flow: column;
+        align-items: center; 
+}
+.cta img{
```

Comment:
The `.cta img { ... }` block may be omitted as well. The img doesn't 'exist' anymore since `display: none` was affects it from the tablet view.